moha/rauk/PariserParr.py

```diff
@@ -251,20 +251,20 @@ def compute_overlap(
                 bond_dictionary[bond_key_forward] = beta_xy
                 bond_dictionary[bond_key_reverse] = beta_xy
         else:
-            for i, tpl in enumerate(connectivity):
+            for tpl in connectivity:
                 atom1, atom2, dist = tpl[0], tpl[1], tpl[2]
```

Comment:
I think we can just unpack tuple as
```suggestion
                atom1, atom2, dist = tpl
```